src/librustc/mir/interpret/mod.rs

```diff
@@ -736,22 +736,23 @@ impl<Tag: Copy, Extra: Default> Allocation<Tag, Extra> {
         &self,
         hdl: impl HasDataLayout,
         offset: Size,
+        size: Size,
+        align: Align,
     ) -> EvalResult<'tcx, Scalar<Tag>> {
```

Comment:
Wait, why does `get_bytes` return a `Scalar<Tag>`? That doesn't make any sense, a method of that name should return a `&[u8]`. Now I am even more puzzled about what it does.

(See, a doc comment would have been useful. ;)

Reply:
this function is called `read_scalar`?

Reply:
Hm did the diff view fool me? Yes it did.

I am still quite irritated why this looks so different from what we do in memory.rs?